watertap/tools/parameter_sweep/parameter_sweep.py

```diff
@@ -305,10 +400,21 @@ def _divide_combinations(self, global_combo_array):
 
         return local_combo_array
 
+    def _get_object(self, model, pyomo_object):
+        name = pyomo_object.name
+
+        # seems to be a bug, as indexed var with [None] exists
+        # but can't be found by find_component
+        if "[None]" in name:
+            name = name.replace("[None]", "")
+            return model.find_component(name)[None]
+        else:
+            return model.find_component(name)
+
     def _update_model_values(self, m, param_dict, values):
         for k, item in enumerate(param_dict.values()):
-            param = item.pyomo_object
-
+            name = self._get_object(m, item.pyomo_object)
+            param = m.find_component(name)
```

Comment:
It seems like `_get_object` returns the component?
```suggestion
            param = self._get_object(m, item.pyomo_object)
```